configure

```diff
@@ -506,7 +506,7 @@ get_system_exe_extension() {
 	riscos)
 		_exeext=",e1f"
 		;;
-	3ds | dreamcast | ds | gamecube | n64 | ps2 | psp | wii)
+	3ds | switch | dreamcast | ds | gamecube | n64 | ps2 | psp | wii)
```

Comment:
These appear to be sorted alphabetically so it should probably be listed after psp?